mission/config/changelog.hpp

```diff
@@ -1,4 +1,16 @@
 class Changelog {
+	class 1_00_03 {
```

Comment:
this should be fine. would probably be good if we can start adding our own changes in here as well.